extensions/gossip/api/gossipapi.go

```diff
@@ -42,3 +42,11 @@ type BlockPublisher interface {
 	// Publish traverses the block and invokes all applicable handlers
 	Publish(block *cb.Block)
 }
+
+// TxnInfo contain txn info
+type TxnInfo struct {
+	BlockNum  uint64
+	ChannelID string
+	TxID      string
```

Comment:
Stutter?

Reply:
(also I see multiple representations of transaction)

Reply:
done